CONTRIBUTING.md

```diff
@@ -55,3 +55,41 @@ Please join our `#socialmedia` channel on Slack! We post amazing content all ove
 [Bee Martinez](https://twitter.com/thebeeletter) is our social media manager. She curates all over our social media channels and will work with you to create your content!
 
 As well as new and original content, we also accept content that has already been published.
+
+
+## Financial contributions
+
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/ifme).
+Anyone can file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
+
+
+## Credits
+
+
+### Contributors
+
+Thank you to all the people who have already contributed to ifme!
+<a href="graphs/contributors"><img src="https://opencollective.com/ifme/contributors.svg?width=890" /></a>
+
+
+### Backers
+
+Thank you to all our backers! [[Become a backer](https://opencollective.com/ifme#backer)]
+
+<a href="https://opencollective.com/ifme#backers" target="_blank"><img src="https://opencollective.com/ifme/backers.svg?width=890"></a>
+
+
+### Sponsors
+
+Thank you to all our sponsors! (please ask your company to also support this open source project by [becoming a sponsor](https://opencollective.com/ifme#sponsor))
+
```

Comment:
Can put a new line here so that the text and images don't look cramped together?